javascript/ql/test/library-tests/StringOps/StartsWith/tst.js

```diff
@@ -31,4 +31,6 @@ function f(A, B) {
 
     if (strings.startsWith(A, B)) {}
     if (strings.caseInsensitiveStartsWith(A, B)) {}
+    if (A.slice(0, B.length) !== B) {}
```

Comment:
Can you also add a path-injection test case that confirms that this also works as a sanitizer?